src/index.js

```diff
@@ -210,6 +213,28 @@ async function Main() {
             });
         });
     }
+
+    async function checkForUpdate () {
+        spinner.start("Checking for an Update...");
+        // Using Github API (https://docs.github.com/en/rest/reference/repos#releases)
+        // to get the releases data
+        const url = "https://api.github.com/repos/vasani-arpit/WBOT/releases";
+        const response = await fetch(url);
+
+        // Storing data in form of JSON
+        var data = await response.json();
+        var latestVersion = data[0].tag_name;
+        var latestVersionLink = `https://github.com/vasani-arpit/WBOT/releases/tag/${latestVersion}`;
+        var myVersion = 'v' + require('../package.json').version;
+
+        spinner.stop("Checking for an Update... Completed");
+
+        if(lt(myVersion, latestVersion)) {
+            console.log(`An Update is available for you.\nPlease download the latest version ${latestVersion} of WBOT from ${latestVersionLink}`);
+        } else {
+            console.log("There is no update available.")
```

Comment:
No need to log anything if there is no update.

Reply:
Updated the PR #281 , Please take a look at #281. (I am messed up with this branch on my local repo that's why I have created a new PR #281 for this same task.)